### PR TITLE
feat: add pino-caller support

### PR DESCRIPTION
**Pull request by almog8k**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✖                                                                       |

Further  information:
Add support of pino-caller.
pino-caller adds the call site of each log message to the log output

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -20,7 +21,9 @@ const jsLogger = (options?: LoggerOptions, destination: string | number = 1): Lo
   }
 
   const pinoOptions: PinoOptions = { ...baseOptions, ...options, transport };
-  return pino(pinoOptions, pino.destination(destination));
+  const logger = pino(pinoOptions, pino.destination(destination));
+  const loggerWithCaller = pinoCaller(logger);
+  return options?.pinoCaller === true ? loggerWithCaller : logger;
```
Comment (CptSchnitz) [inline src/index.ts:26]:
```suggestion
  return options?.pinoCaller === true ? pinoCaller(logger) : logger;
```
i think running less code > clarity in this case.
what do you think?

**Review by CptSchnitz — CHANGES_REQUESTED** [review]
READMEEEEEEEE (add a line plz)

**Comment by almog8k** [comment]
> READMEEEEEEEE (add a line plz)

Added

**Review by CptSchnitz — APPROVED** [review]

File: src/index.ts (hunk 1 above)
Comment (almog8k) [reply src/index.ts]: i agree

**Pull request merged**